reference/performance/execution-plan-bind.md

```diff
@@ -56,6 +82,20 @@ DROP [GLOBAL | SESSION] BINDING FOR SelectStmt
 
 This statement removes a specified execution plan binding at the GLOBAL or SESSION level. The default scope is SESSION.
 
+Generally, the binding in the SESSION scope is mainly used for test or in special situations. For a binding to take effect in all TiDB processes, you need to use the GLOBAL binding. The SESSION binding overrides the GLOBAL binding until the end of the SESSION.
```

Comment:
This may sound clearer.
```suggestion
Generally, the binding in the SESSION scope is mainly used for test or in special situations. For a binding to take effect in all TiDB processes, you need to use the GLOBAL binding.  A created SESSION binding will overrides the corresponding GLOBAL binding until the end of the SESSION, even if the SESSION binding is dropped before the session closes. (in this case, no binding will take effect and the plan will be selected by the optimizer).
```

Reply:
Shield can be used to replace overrides. rest LGTM

Reply:
Thanks! Suggestion applied.